docs/roles/secretary.rst

```diff
@@ -9,22 +9,33 @@ This page explains more about the responsibilities and role of the secretary.
 Description
 -----------
 
-< general description of role, 1-2 paragraphs >
-
+The secretary is responsible for scheduling, advertising, and recording RITlug general and eboard meetings.
 
 Eligibility requirements
 ^^^^^^^^^^^^^^^^^^^^^^^^
 
-< basic eligiblity requirements – what if someone is on co-op? away from campus? working remote? is under academic suspension (i.e. low GPA)? >
+* Must be a full time student on main campus for at least one semester
+* Cannot be on academic probation
+* Must be present for **75% of weekly club meetings and eboard meetings**
 
 
 Responsibilities
 ----------------
 
-< responsibilities and routine work that someone in this position is directly accountable for – what do they do and what should other eboard members expect from this person? >
+* *Weekly Presentations*
+   * Review of presentations
+   * Addition of presentations to RITlug website
+* Record meeting minutes during both general and eboard meetings
+* Setup live stream or video conference for meetings and presentations that require remote participants or viewers
+* Take attendance of both general and eboard meetings
```

Comment:
This is done by the sign-in form already. Perhaps a better rewrite to this is taking the responsibility to set up the sign-in form for upcoming meetings?

Reply:
Good point, since attendance is done through campus groups for the general meeting. Eboard attendance is just typically recorded in the minutes, right?

Reply:
Typically, yes. However, we don't usually keep track of attendance at the eboard meetings in any viewable location irrc.